extras/mainnet.json

```diff
@@ -153,6 +154,9 @@
         "LenderVaultFactory": "0x1874a08f7975b25944feb989bbaaa464f61ab3bc"
 
     },
+    "safe": {
+        "token_locker": "0x0a7CB434f96f65972D46A5c1A64a9654dC9959b2"
```

Comment:
Anyway to verify this address? I don't see many transactions on it and it doesn't pop up in docs when I google it.

Reply:
https://github.com/safe-global/safe-locking?tab=readme-ov-file#mainnet-deployment